src/task/MonitorTask.ts

```diff
@@ -200,6 +203,8 @@ export default class MonitorTask
       StreamMetricReport
     >();
 
+    this.videoEncodingMonitor.encodingMonitor(clientMetricReport);
```

Comment:
nit: maybe this function can just be named `metricsDidReceive` as well. Maybe in the future we can make that its own interface.

Reply:
Removed previous class and use `metricsDidReceive` in connection monitor to update metrics.